.gitignore

```diff
@@ -4,4 +4,6 @@ package-lock.json
 .DS_Store
 yarn.lock
 release
-_secrets
\ No newline at end of file
+_secrets
+.eslintrc.js 
```

Comment:
y?????

Reply:
What is the difference between eslintrc with a .json extension vs .js

Reply:
And why is one gitignored. Doesn’t this kind of defeat the purpose of a game eslintrc ... standardizing style for all devs working in the codebase

Reply:
oh nvm they're using a file just not using the one we made

Reply:
Now that we are at the end of development we re-ran the code through the original linter and cleaned up various aspects of the code.